docs/content.md

```diff
@@ -0,0 +1,43 @@
+# MITRE ATT&CK Threat Hunting
+
+# Overview
+
+The MITRE ATT&CK Threat Hunting Solution Pack demonstrates a variety of scenarios and use cases around threat hunting using the information provided by the MITRE ATT&CK Framework. 
+
+*MITRE ATT&CK is a globally-accessible knowledge base of adversary tactics and techniques based on real-world observations. The ATT&CK knowledge base issued as a foundation for the development of specific threat models and methodologies in the private sector, in government, and in the cybersecurity product and service community.*
+
+## Pre-requisite
+
+### Solution Packs
+
+This Solution Pack requires the deployment of the following solution packs.
+
+<table>
+    <thead>
+        <tr>
+            <th>Solution Pack</th>
+            <th>Purpose</th>
```

Comment:
This is diff formatting other than pipes which we used else where... what is a reason for not using pipe here?

Reply:
removed